docs/architecture/decisions/0004-what-formats-to-use-for-git-tag.md

```diff
@@ -0,0 +1,26 @@
+# 4. What formats to use for Git tag
+
+Date: 2022-09-16
+
+## Status
+
+Proposed
+
+## Context
+
+We use Git tags to mark project versions and those tags are used to build the container image. 
```

Comment:
@lucacavallaro do you suggest to use git commit sha instead?

Reply:
We will use the procedure explained in the ADR 5 to change the project version, so the relationship between the tag and the `package.json` version is 1:1: the command to update the project's version also create the Git tag, so they are aligned.